versioned_docs/version-v5/developing/android.md

```diff
@@ -153,7 +153,7 @@ Capacitor uses Android Studio to build and run apps to simulators and devices.
 
 2. **In Android Studio, click the Run button and then select the target simulator or device.**
 
-![Android Studio Run Button Area](/img/running/android-studio-run-button-area.png)
+![Screenshot highlighting the run button area in Android Studio to build and run an Android app.](/img/running/android-studio-run-button-area.png 'Android Studio Run Button Area')
```

Comment:
This one also needs to updated to exclude "screenshot". The same goes for the same sentences found in other files.